integration-libs/punchout/core/facade/punchout.service.ts

```diff
@@ -49,6 +49,18 @@ export class PunchoutService implements PunchoutFacade {
   protected multiCartFacade = inject(MultiCartFacade);
   protected userIdService = inject(UserIdService);
 
+  /**
+   * punchoutSession ajax request will always return same response during a punchout session.
+   * To avoid multiple server calls, input (punchoutSessionId) and output (punchoutSession) values are stored into punchoutSessionRequest.
+   */
+  protected readonly punchoutSessionRequest: {
```

Comment:
could be rename as punchoutSessionRequestSnapshot, cachedPunchoutSession or punchoutSessionRequestState if not explicit enough.

Reply:
I realized we don't need to cache the sessionId (payload) since there is no scenario where sessionId would be different during a user punchout instance without leaving spartacus.
so now only caching the response:
` protected cachedPunchoutSessionResponse?: PunchoutSession;`